### PR TITLE
Training notebook listings

### PR DESCRIPTION
**Pull request by lesserwhirls**
Create a list of the training notebooks that are needed (site/resources.md), and add them to the appropriate project page. As notebooks are created, links can be added to the site.

### CODE REVIEW

File: site/projects/isentropic_analysis.md [hunk 1]
```diff
@@ -10,16 +10,29 @@ _description here_
...
+* Siphon (catalogs basic)
+* Siphon (catalogs filtering)
+* Siphon (remote_access)
+* Siphon (remote_open)
+* Siphon (subset)
 
 ### Analysis
-* xarray basics (data access, indexing, and interpolation)
-* xarray intermediate (aggregations and calculations)
+* xarray  (data access)
```
Comment (dopplershift) [inline site/projects/isentropic_analysis.md:21]: Extra space after xarray

File: site/projects/model_vs_obs.md [hunk 2]
```diff
@@ -12,20 +12,26 @@ Highlight ways to enhance timeseries charts (time formatting on x-axis, latex la
...
-* xarray basics (data access)
-* python-awips model
-* python-awips surface obs
+* Siphon (TDS intro)
+* Siphon (catalogs basic)
+* Siphon (catalogs filtering)
+* Siphon (remote_access)
+* Siphon (remote_open)
+* Siphon (subset)
+* xarray  (data access)
```
Comment (dopplershift) [inline site/projects/model_vs_obs.md:21]: Extra space after xarray

File: site/projects/rossby_wave_analysis.md [hunk 3]
```diff
@@ -11,14 +11,17 @@ Allows application of existing calculations and implementation of own calculatio
 ## Related Training Notebooks
 
 ### Data Access
-* xarray basics (data access)
+* xarray  (data access)
```
Comment (dopplershift) [inline site/projects/rossby_wave_analysis.md:14]: Here too

File: site/projects/severe_wx_outbreak.md [hunk 4]
```diff
@@ -10,19 +10,28 @@ _description here_
...
 ### Data Access
-* Siphon
-* xarray basics (data access)
+* Siphon (TDS intro)
+* Siphon (catalogs basic)
+* Siphon (catalogs filtering)
+* Siphon (remote_access)
+* Siphon (remote_open)
+* Siphon (subset)
+* xarray  (data access)
```
Comment (dopplershift) [inline site/projects/severe_wx_outbreak.md:19]: And here

**Review by dopplershift — APPROVED** [review]
Just some extra spaces--just let me know if those aren't worth fixing because they'll go away when things get filled out.

File: site/projects/isentropic_analysis.md (hunk 1 above)
Comment (lesserwhirls) [reply site/projects/isentropic_analysis.md]: "find" + "replace all" is good, when the replace with is correct and does not contain extra spaces! 😆 Fixed now.

**Pull request merged**